tests/test_iri_resolver.py

```diff
@@ -275,6 +276,46 @@ def test_relative_with_triple_dot_segment_and_2x_double_dot_and_base(self):
     def test_questionmark_prefix_relative_with_complex_base_with_dot(self):
         assert resolve('?y','http://a/bb/ccc/./d;p?q') == 'http://a/bb/ccc/./d;p?y'
 
+# ---------- Tests for unresolve() ----------
+class TestUnresolve:
```

Comment:
Should class-based or function-based `pytest` testcases be chosen as uniform practice for this project?